lib/service_sign_in_yaml_validator.rb

```diff
@@ -48,13 +48,28 @@ def check_for_choose_sign_in_required_fields
       return
     end
 
+    check_choose_sign_in_top_level_fields(error_message)
+    return unless choose_sign_in["options"].present?
```

Comment:
Shouldn't this line return an error as there must be at least one (two?) options?

Reply:
`check_choose_sign_in_top_level_fields` will check if `options` is present and log an error if it isn't.  However, we should probably check if `options` has an array of options, similar to what we do in line 46.

Reply:
We don't need the guard then, as "options" must be present at this point. But you're right, we should check that options has an array.

Reply:
If `options` isn't present that we don't want to call `check_choose_sign_in_options_fields` as it'll blow up, so we return in this scenario.
Going to add in a check that if `options` is present then it's an array 👍